pkg/util/hostname.go

```diff
@@ -229,7 +229,7 @@ func GetHostname() (string, error) {
 	if err == nil && !config.Datadog.GetBool("hostname_fqdn") && fqdn != "" && hostName == h && h != fqdn {
 		if runtime.GOOS != "windows" {
 			// REMOVEME: This should be removed in 6.7
```

Comment:
`REMOVEME` needs an update as well, this should be removed when the default is switched to `true`